packages/catalog-process/src/services/catalogService.ts

```diff
@@ -653,7 +653,10 @@ export function catalogServiceBuilder(
 
       if (
         document.kind === "DOCUMENT" &&
-        descriptor.docs.some((d) => d.prettyName === document.prettyName)
+        descriptor.docs.some(
+          (d) =>
+            d.prettyName.toLowerCase() === document.prettyName.toLowerCase()
+        )
```

Comment:
Even though the Jira task did not mention it, I made case-insensitive the check on documents as well, does it make sense?

Reply:
@taglioni-r  what do you think? I'm asking you as the reporter of the task on Jira!
After this PR, I will do the same in the rest of the  `*-process` services

Reply:
From my point of view it make sense to make also the doc name check case insensitive

Reply:
@taglioni-r what do you think? I'm asking you as the reporter of the task on Jira!
After this PR, I will do the same in the rest of the *-process services